common/src/main/java/org/apache/uniffle/common/config/RssClientConf.java

```diff
@@ -271,6 +272,19 @@ public class RssClientConf {
           .withDescription(
               "Whether to support rss client block send failure retry, default value is false.");
 
+  public static final ConfigOption<PartitionSplitMode> RSS_CLIENT_PARTITION_SPLIT_MODE =
+      ConfigOptions.key("rss.client.reassign.partitionSplitMode")
+          .enumType(PartitionSplitMode.class)
+          .defaultValue(PartitionSplitMode.PIPELINE)
+          .withDescription("The partition split mode. default is PIPELINE.");
+
+  public static final ConfigOption<Integer> RSS_CLIENT_PARTITION_SPLIT_LOAD_BALANCE_SERVER_NUMBER =
+      ConfigOptions.key("rss.client.reassign.partitionSplitLoadBalanceServerNumber")
+          .intType()
+          .defaultValue(10)
```

Comment:
The default `10` maybe too big for the small cluster, is there a way to define a ratio of cluster's worker number?
For example, 0.5 means half of works.

Reply:
sounds good, but this will refactor the coordinator rpc. So I will remain this optimization into another PR.